extensions/amp-render/1.0/amp-render.js

```diff
@@ -116,15 +115,6 @@ export class AmpRender extends BaseElement {
     this.src_ = null;
   }
 
-  /** @override */
-  isLayoutSupported(layout) {
-    userAssert(
-      isExperimentOn(this.win, 'amp-render'),
-      'Experiment "amp-render" is not turned on.'
-    );
```

Comment:
Is this removal intended? Since there are validations, removing this experiment assertion will affectively launch the component.

Reply:
Yes, this is intended. Based on conversation with @nainar & @alanorozco, components launched behind an experiment typically don't get high usage, while we want this to get usage and collect feedback.